resources/careconnect-to-ukcore/allergyintolerance/maps/AllergyIntoleranceExtension-coding-sctdescid-3to4.map

```diff
@@ -19,7 +19,7 @@ group Coding(source src : Coding, target tgt : Coding) <<type+>> {
     src.userSelected as userSelected -> tgt.userSelected = userSelected;
 
     src.extension as vs where $this.url='https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-coding-sctdescid' or 
-                              this.url='https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid' ->
+                              $this.url='https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid' ->
```

Comment:
This had also cause an issue, no error was thrown, but the validator never used this statement in the where clause, so extensions with the url 'https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid' weren't being converted, should be fixed in last commit